src/StreamsAPI_Lambda/ArrayList/FirstMethod.java

```diff
@@ -0,0 +1,23 @@
+package StreamsAPI_Lambda.ArrayList;
```

Comment:
I'm leaving it here but it applies to all the classes: package names are lower case, see the [naming conventions in Java](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html)